cirq/circuits/qasm_output.py

```diff
@@ -167,6 +167,18 @@ def __init__(
         precision: int = 10,
         version: str = '2.0',
     ) -> None:
+        """Representation of a circuit in QASM format.
+
+        Args:
+            operations: Tree of operations to insert.
+            qubits: The qubits used in the operations.
+            header: A multi-line string that is placed in a comment at the top
+                of the QASM. Defaults to a cirq version specifier.
```

Comment:
> Defaults to a cirq version specifier.

What does this mean? The default is an empty string.: `header: str = '',`

Reply:
You are right, my bad. Should it be something like _Defaults to an empty string._, or would it be better to just leave it without that part?

Reply:
Let's leave it out.

Reply:
Done.